### PR TITLE
Remove event table

### LINKED ISSUE #518: reduce amount of logging
Currently, `event_log` is getting pretty big.  Consider cutting back / making this more efficient.

### PR DESCRIPTION
**Pull request by candicegjing**
Fixed #518 .

### CODE REVIEW

**Review by percyliang — APPROVED** [review]

**Comment by percyliang** [comment]
Just make sure tests pass.

File: codalab/server/rest_server.py [hunk 1]
```diff
@@ -27,28 +26,12 @@
 
 from codalab.common import exception_to_http_error
 from codalab.lib import formatting, server_util
-import codalab.rest.account
```
Comment (epicfaace) [inline codalab/server/rest_server.py:31]: Perhaps this change is what is causing the tests to fail -- maybe you need these imports in order to add the routes?

Comment (candicegjing) [reply codalab/server/rest_server.py:31]: Hmm, let me check, somehow my IDE shows they are unused imports. I think you are right. Adding them back makes the build pass.

Comment (candicegjing) [reply codalab/server/rest_server.py]: @epicfaace I think you are right. Adding them back makes the build working. I am just curious how are these imports get called in this code...

**Comment by percyliang** [comment]
Can we put a comment on that line that says don't remove for now?  We
should figure out how to make this so subtle/confusing.

**Comment by candicegjing** [comment]
I have added a comment for those imports. I was also trying to debug why we need those imports given nothing is calling them directly...

**Review by epicfaace — APPROVED** [review]
Any more changes needed before we can merge this?

**Pull request merged**